CHANGELOG.md

```diff
@@ -55,7 +55,7 @@ Unreleased
 
 #### Changed
 
--   Switched to ISO 639-3 language codes. (\#468)
+-   Removed `casefold:true` from langauges.json and rescraped `hye` and `apw` (\#469)
```

Comment:
* The new entry says `rescraped hye and apw`, but the new data isn't included in this PR?
* Why remove the existing entry `Switched to ISO 639-3 language codes. (\#468)`